### PR TITLE
Fix/sniff ecms wcm

### PR DESCRIPTION
**Pull request by ilyeskallala**

### CODE REVIEW

File: gatein/src/main/java/org/exoplatform/platform/qa/ui/gatein/pageobject/ContentList.java [hunk 1]
```diff
@@ -28,15 +32,17 @@ public ContentList(TestBase testBase) {
...
-      for (String arrayElement : arrayPath) {
-        $(byText(arrayElement)).click();
-      }
+    for (String arrayElement : arrayPath) {
+      $(byXpath(ELEMENT_MULTIPLE_CONTENT_POPUP_NODE_FOLDER.replace("${node}", arrayElement))).click();
     }
 
+    if (content != "" || content != null) {
+      $(byXpath(ELEMENT_MULTIPLE_CONTENT_POPUP_FILE.replace("${content}", content))).waitUntil(Condition.visible,Configuration.timeout);
+      //click(ELEMENT_MULTIPLE_CONTENT_POPUP_FILE.replace("${content}",content));
```
Comment (thomasdelhomenie) [inline gatein/src/main/java/org/exoplatform/platform/qa/ui/gatein/pageobject/ContentList.java]: Commented code to remove.

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsWCMTestIT.java [hunk 2]
```diff
@@ -0,0 +1,714 @@
...
+import org.exoplatform.platform.qa.ui.ecms.pageobject.SiteExplorerHome;
+import org.exoplatform.platform.qa.ui.gatein.pageobject.*;
+import org.exoplatform.platform.qa.ui.selenium.platform.HomePagePlatform;
+import org.exoplatform.platform.qa.ui.selenium.platform.ManageLogInOut;
+import org.exoplatform.platform.qa.ui.selenium.platform.NavigationToolbar;
+import org.exoplatform.platform.qa.ui.selenium.platform.administration.ChangeLanguages;
+
+/**
+ * @date 15-January-2015
+ * @author exo update 26/01/2015
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsWCMTestIT.java]: To remove

File: gatein/src/main/java/org/exoplatform/platform/qa/ui/gatein/pageobject/ContentList.java (hunk 1 above)
Comment (aboughdiri) [reply gatein/src/main/java/org/exoplatform/platform/qa/ui/gatein/pageobject/ContentList.java]: Done

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsWCMTestIT.java (hunk 2 above)
Comment (aboughdiri) [reply platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsWCMTestIT.java]: Done

**Pull request merged**